backend/connector-integration/src/connectors/getnet/transformers.rs

```diff
@@ -178,9 +202,10 @@ impl<T: PaymentMethodDataTypes + std::fmt::Debug + Sync + Send + 'static + Seria
         let card_data = match &item.request.payment_method_data {
             PaymentMethodData::Card(card) => card,
             _ => {
-                return Err(errors::ConnectorError::NotImplemented(
-                    "Payment method not supported".to_string(),
-                )
+                return Err(errors::ConnectorError::NotSupported {
+                    message: "Payment method not supported".to_string(),
```

Comment:
```suggestion
                    message: "Payment method ".to_string(),
```
The current implementation would serialize to ```Payment method not supported is not supported by Getnet ```